.github/workflows/stale.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: Mark and close stale issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
```

Comment:
Nit: Suggest we pin version by hash as we do for other github actions for security reasons.

Reply:
Done.